io-sim/src/Control/Monad/IOSim/Internal.hs

```diff
@@ -1278,12 +1288,14 @@ data StmStack s b a where
                    -> StmStack s b c
                    -> StmStack s a c
 
-execAtomically :: forall s a.
-                  TVarId
+execAtomically :: forall s a c.
+                  Time -> ThreadId -> Maybe ThreadLabel
```

Comment:
I'd prefer to break it to get one arg per line.

Reply:
Fixed, thanks for catching my sloppiness there.